pkg/cloud/services/networking/port.go

```diff
@@ -179,7 +179,6 @@ func (s *Service) GetOrCreatePort(eventObject runtime.Object, clusterName string
 	if portOpts.Trunk != nil && *portOpts.Trunk {
 		trunk, err := s.getOrCreateTrunk(eventObject, clusterName, port.Name, port.ID)
 		if err != nil {
-			record.Warnf(eventObject, "FailedCreateTrunk", "Failed to create trunk for port %s: %v", portName, err)
```

Comment:
I was thinking keep this one and remove the events in getOrCreateTrunk()? By emitting the event at both error points we're no longer avoiding duplicating the error event.

Reply:
Yeah, I'd probably go with what Matt says here, but no strong preference.